Fixing issue 100 - Exception: Serialization of 'Closure' is not allow…

patriziotomato opened the pull request:
…ed #100

WendellAdriel commented:
@patriziotomato changes seem ok to me! ✅ 
Thanks for the help on that! 🔥 
Just make sure to run the linter so the checks can pass! 💪

patriziotomato commented:
@WendellAdriel Did run `docker compose run --rm app composer lint` but it hasn't changed any file. Any idea why?

WendellAdriel commented:
> @WendellAdriel Did run `docker compose run --rm app composer lint` but it hasn't changed any file. Any idea why?

That's interesting, can you just try one more thing, if that doesn't work I'll accept the PR and run it on my end.
Try removing your vendor folder and running the `make install` command

patriziotomato commented:
> > @WendellAdriel Did run `docker compose run --rm app composer lint` but it hasn't changed any file. Any idea why?
> 
> That's interesting, can you just try one more thing, if that doesn't work I'll accept the PR and run it on my end. Try removing your vendor folder and running the `make install` command

That made it. Not sure if you want to have this pushed into that project, but I added it to my PR.

I also added a test now and still don't get it, why here my linter is just fine, while it fails in Github Actions. Any idea for hat @WendellAdriel  ?

WendellAdriel commented:
> > > @WendellAdriel Did run `docker compose run --rm app composer lint` but it hasn't changed any file. Any idea why?
> > 
> > 
> > That's interesting, can you just try one more thing, if that doesn't work I'll accept the PR and run it on my end. Try removing your vendor folder and running the `make install` command
> 
> That made it. Not sure if you want to have this pushed into that project, but I added it to my PR.
> 
> I also added a test now and still don't get it, why here my linter is just fine, while it fails in Github Actions. Any idea for hat @WendellAdriel ?

Have you tried running the `rm -rf vendor && make install` command to see if it changes anything for the linter?

WendellAdriel commented:
@patriziotomato can you update the PR to be updated with the latest version released, please?

patriziotomato commented:
> docker compose run --rm app composer lint

Unfortunately still same result. I rebased the branch, so that it is up-to-date...

WendellAdriel commented:
> > docker compose run --rm app composer lint
> 
> Unfortunately still same result. I rebased the branch, so that it is up-to-date...

Weird that it seems to be running the "old" actions because I dropped the support and tests from Laravel 9 and 10 and PHP 8.1 and it's still showing those runs for the actions. 😢 

Can you try to create a new branch from the main one and create a new PR with the changes, please?
